CMakeLists.txt

```diff
@@ -448,6 +436,28 @@ message(STATUS "  Override with -DENABLE_INTERNAL_VOLK=ON/OFF")
 # Handle logging
 find_package(LOG4CPP REQUIRED)
 
+########################################################################
```

Comment:
Guessing this clause is the conflict in this file.

Reply:
This should be resolved with the lastest push